MaxText/configs/base.yml

```diff
@@ -590,7 +590,7 @@ jax_profiler_port: 9999
 log_config: True # Prints the config (after defaults have been set by pyconfig logic)
 
 # Checkpoint Structured logging
-enable_checkpoint_cloud_logger: False
+enable_checkpoint_cloud_logger: True
```

Comment:
Why do we need to change the default here?

Reply:
We don't need it to be default, removed this file and squashed commits.

Reply:
Yep, this is the same issue. For Goodput, the temporary solution is in https://b.corp.google.com/issues/401509894#comment7 until this bug is resolved.